src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
Don't create temporary variables if you can directly return value from the method. Instead of creating a `ball` variable, you can directly return the new `Ball` object after setting its color and number.